src/features/stats/index.js

```diff
@@ -4,6 +4,7 @@ import ReactTimeout from 'react-timeout';
 
 import { ANIMATION_SPEED } from '../../config/constants';
 import calculateModifier from '../../utils/calculate-modifier';
+import calculateMaxManapool from '../../utils/calculate-manapool-intelligence';
```

Comment:
Not blocking, but 'mana' and 'pool' are separate words so should be cased separately i.e. `calculateMaxManaPool`, although that being said I don't think 'pool' is even necessary.
Also, I think `calculate-manapool-intelligence` could be renamed to be shorter without losing any description. Again, these aren't blocking changes, so feel free not to implement them.

Reply:
Alright, I'll go change it.